redshift_connector/cursor.py

```diff
@@ -239,6 +238,87 @@ def executemany(self: "Cursor", operation, param_sets) -> "Cursor":
         self._row_count = -1 if -1 in rowcounts else sum(rowcounts)
         return self
 
+    def insert_data_bulk(self: "Cursor", filename,table_name, column_indexes,column_names,delimeter)-> "Cursor":
+  
+        """runs a single bulk insert statement into the database.
+           
+           This method is native to redshift_connector.
+
+            :param filename: str
+                The name of the file to read from.
+            :param table_name: str
+                The name of the table to insert to.
+            :param column_names:list
+                The name of the columns in the table to insert to.  
+            :param column_indexes:list
+                The indexes of the columns in the table to insert to.
+            :param delimeter: str
+                The delimeter to use when reading the file.
+
+            Returns
+            
+            -------
+            The Cursor object used for executing the specified database operation: :class:`Cursor`
+
+        """
+        if not self.__is_valid_table(table_name):
+            raise InterfaceError("Invalid table name passed to insert_data_bulk: {}".format(table_name))
+        if not self.__has_valid_columns(table_name, column_names):
+            raise InterfaceError("Invalid column names passed to insert_data_bulk: {}".format(table_name))
+        try:
+            import csv
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(MISSING_MODULE_ERROR_MSG.format(module="csv"))
+        if len(column_names)!=len(column_indexes):
+            raise InterfaceError("Column names and indexes must be the same length")
+        sql_query = f"INSERT INTO  {table_name} ("
+        for column_name in column_names:
+            sql_query = sql_query + f", {column_name}"
```

Comment:
```suggestion
sql_query += ','.join(column_names)
```

Otherwise, the existing code prepends each column name with a comma. e.g. `INSERT INTO my_table (, col1`